MbuguaCaleb/Questioner-endpoints
# [finishes#163073666] making the directory structure

**What does this PR do?**

Creates the directory structure with the necessary files and packages for running the app.

**Description of Tasks completed ??**

Sets up the folder /package structure for versioning my apis and running  packages
adds and init.py file to each module

**How should this be manually tested?**

Clone this repository on your machine:
git clone https://github.com/MbuguaCaleb/Questioner-endpoints.git

CD into the project folder on your machine
$ cd Questioner-endpoints

Check out the branch for this PR
$ git checkout  ch-directory-structur-163073666

****Relevant pivotal tracker story associated with this PR**
#163073666

**screenshots**
![screenshot from 2019-01-09 04-28-07](https://user-images.githubusercontent.com/45809091/50869912-0e3fe980-13c7-11e9-9e34-74e36ee8765b.png)
